tests/test_telegram.py

```diff
@@ -0,0 +1,130 @@
+from fastapi import status
+import pytest
+
+from app.telegram.handlers import MessageHandler, reply_unknown_user
+from app.telegram.models import Bot, Chat
+
+
+def gen_message(text):
+    return {
+        'update_id': 10000000,
+        'message': {
+            'message_id': 2434,
+            'from': {
+                'id': 666666,
+                'is_bot': False,
+                'first_name': 'Moshe',
+                'username': 'banana',
+                'language_code': 'en'
+            },
+            'chat': {
+                'id': 666666,
+                'first_name': 'Moshe',
+                'username': 'banana',
+                'type': 'private'
+            },
+            'date': 1611240725,
+            'text': f'{text}'
+        }
+    }
+
+
+def test_chat_model():
```

Comment:
Consider surround tests with a class.
Also, you can add some consts (like "not found") and use it in the tests.